example/src/App.tsx

```diff
@@ -113,33 +114,33 @@ export default function App() {
 
   return (
     <View style={styles.container}>
-      <Text>Room name:</Text>
+      <Text style={Fonts.Headlines.h5}>Room name:</Text>
```

Comment:
I'd prefer to have our custom styled `<Text>` component with a prop like this:
`<Text variant="h5">Room name:</Text>`
Or you can name it `<Typo>` to make it less confusing with standard RN Text. 
That way it's less typing and no need to include styles and when we want to change all texts in the app we'd need to do it in one place.
Also make it typed so that vscode will suggest and check text variant.
Other components can be done in a similar way.

Reply:
SG, I'll do that.